test/server/cards/01_SOR/events/StrikeTrue.spec.ts

```diff
@@ -0,0 +1,66 @@
+describe('Strike True', function() {
+    integration(function(contextRef) {
+        describe('Strike True\'s ability', function() {
+            beforeEach(function () {
+                contextRef.setupTest({
+                    phase: 'action',
+                    player1: {
+                        hand: ['strike-true'],
+                        groundArena: ['wampa', 'battlefield-marine']
+                    },
+                    player2: {
+                        groundArena: ['specforce-soldier', 'atst'],
+                        spaceArena: ['tieln-fighter']
+                    }
+                });
+            });
+
+            it('should deal damage to an enemy unit equals to a friendly unit power', function () {
```

Comment:
I think we're okay without this test, it's more testing that the defeat due to damage works correctly than validating the implementation of the card. We do mix tests like that in this repo sometimes but in this case there's already a ton of tests validating this interaction.

I'd rather keep the below test where the unit survives so we can validate that the amount is correct (can't do that if the unit is defeated obviously)

Reply:
sounds good, I just removed it